apis/eseries/eseries.go

```diff
@@ -111,7 +118,9 @@ func (d Driver) SendMsg(data []byte, sendType string, msgType string) (*http.Res
 	req.SetBasicAuth(d.config.Username, d.config.Password)
 
 	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: d.config.WebProxyInsecure,
```

Comment:
Should we have two settings then? One for controlling http vs https and another for skipping TLS verification? 

I'd lean towards the following:
- webProxyUseHttp (default false)
- webProxySkipTLSVerify (default false)